### ISSUE TITLE
MDClone Data: clean & prep for modeling

### ISSUE THREAD

**Issue by lorikrammer** [GW-HIVE/PredictMod#90]

**Comment by lorikrammer** [comment]
Contingent upon completion of #86.

**Comment by lorikrammer** [comment]
I followed up today with the COR and VA IT help and I am still waiting on a final approval from the VA to obtain access to MDClone. Once the approval goes through, it should take about 20 minutes to reflect in the system and I should be able to begin working within MDClone at that point. The in-progress query has been shared with me so it will also be ready to use as soon as I have access.

**Comment by lorikrammer** [comment]
Instructions to access MDClone via the Citrix Access Gateway (temporary PIV exemption).

- Go to [https://citrixaccess.va.gov/](https://citrixaccess.va.gov/). Use the temporary credentials provided by VA IT to login. 
- Use the 1VA-Virtual Desktop by clicking on the icon available at the home page. 
- Once the virtual desktop is loaded, open a browser and go to [spd.vha.med.va.gov/datalake/#/queries](http://spd.vha.med.va.gov/datalake/#/queries).

**Comment by lorikrammer** [comment]
Citrix Contact Info: Shauna at the VA: 708-483-5202, Ticket #INC28514014, Central Time

**Comment by lorikrammer** [comment]
Called and left a message with Shauna at Citrix. I will try again tomorrow morning.

**Comment by lorikrammer** [comment]
I spoke with VA IT yesterday (8/30) and they extended my PIV exemption through 9/6. However, when trying to use that login, the credentials were not recognized. During the call I was unable to use my prior login credentials. After the call with IT, I tried again and I am still able to log in with my previous PIV exemption credentials (scheduled to expire 8/31). VA IT also put in a ticket for Citrix since the login credentials were not working. Based on their feedback, I will need to request access to Citrix Access Gateway, since it did not show that I have access to that VPN. We tried accessing through Cisco with no luck.

**Comment by lorikrammer** [comment]
**08.30.2023**
I access MDClone and completed the mini-query with 7 data properties of interest. I was able to extract this query but since it is within the VA virtual desktop and I am still unable to access my VA email, I do not have a way to pull the .csv file out of the VA environment. I will reach out to VA IT tomorrow morning to a) confirm my exemption credentials extension, and b) access outlook, submit the citrix access gateway request for authorization. 

In the meantime, I have sent a copy of the query design to Vinod, so that during the 9/1 meeting we can review it from his side at the very least.

**Comment by lorikrammer** [comment]
I met with Vinod today and was able to make some progress on the MDClone query:

- We began review of the mini-query and confirmed the intervention, inclusion criteria, and outcome measures were correctly included. 
- To best capture the data for each variable of interest, we need to compare a LOINC code search with a keyword search to increase the data collected. Vinod provided a link to VA's CIPHER resource, where we can review the search logic used.
- Vinod walked me through this process for an example variable, HbA1c. We were able to increase the HbA1c results from 305 included terms to 372. 

I will work with Urnisha to finish the remaining variables following this process. I set a meeting for 9/29 @ 2:00 PM ET with Vinod. If we finish before that date or have any questions for Vinod, we will reach out to him. 

I am still working with VA's IT to obtain permanent access using my PIV card. In the meantime, I am using PIV exemption credentials to continue working with MDClone.

**Comment by lorikrammer** [comment]
Enterprise Service Desk (ESD) at 855-673-4357 (TTY: 1 844 224 6186)

**Comment by lorikrammer** [comment]
I spoke with Enterprise IT (Ticket #INC28817981) and after troubleshooting, they put in a request for the Local IT to contact me. I provided my phone number as well as my email address (lorikrammer@gwu.edu). They will see if they can identify the issue on their end, and determine if an onsite visit or new card is needed.

**Comment by lorikrammer** [comment]
@AbelArgaw Are you able to access MDClone at this link? [spd.vha.med.va.gov/datalake/#/queries](http://spd.vha.med.va.gov/datalake/#/queries).
